dll/win32/shell32/dialogs/drive.cpp

```diff
@@ -166,9 +166,20 @@ typedef struct _DRIVE_PROP_PAGE
     UINT DriveType;
 } DRIVE_PROP_PAGE;
 
-BOOL
-SH_ShowDriveProperties(WCHAR *pwszDrive, IDataObject *pDataObj)
+struct DRIVE_PROP_DATA
+{
+    LPWSTR pwszDrive;
+    IDataObject *pDataObj;
+};
+
+static DWORD WINAPI
+SH_ShowDrivePropThreadProc(LPVOID pParam)
```

Comment:
This is not a public helper function, it does not need a SH prefix.

Reply:
Fixed.